substrate/client/transaction-pool/tests/zombienet/mod.rs

```diff
@@ -0,0 +1,161 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! The zombienet spawner for integration tests for a transaction pool. Holds shared logic used
+//! across integration tests for transaction pool.
+
+use anyhow::anyhow;
+use tracing_subscriber::EnvFilter;
+use txtesttool::scenario::{ChainType, ScenarioBuilder};
+use zombienet_sdk::{
+	subxt::SubstrateConfig, LocalFileSystem, Network, NetworkConfig, NetworkConfigExt,
+};
+
+/// Gathers network TOML specifications file paths.
+pub mod asset_hub_based_network_spec_paths {
+	pub const LOW_POOL_LIMIT_FATP: &'static str =
+		"tests/zombienet/network-specs/asset-hub-low-pool-limit-fatp.toml";
+	pub const HIGH_POOL_LIMIT_FATP: &'static str =
+		"tests/zombienet/network-specs/asset-hub-high-pool-limit-fatp.toml";
+	pub const HIGH_POOL_LIMIT_FATP_RELAYCHAIN: &'static str =
+		"tests/zombienet/network-specs/asset-hub-high-pool-limit-fatp-relaychain.toml";
```

Comment:
we may need new mod for this - `rococo_based_...`?

Reply:
Ah, right. Will change

Reply:
Done here: [f2409a9](https://github.com/paritytech/polkadot-sdk/pull/7257/commits/f2409a9a22f7893d8cb1dda64bc47c9fca9872b8)